tests/test_wave_05.py

```diff
@@ -113,18 +118,15 @@ def test_delete_goal(client, one_goal):
     response = client.get("/goals/1")
     assert response.status_code == 404
 
-@pytest.mark.skip(reason="test to be completed by student")
 def test_delete_goal_not_found(client):
-    pass
-
     # Act
-    # ---- Complete Act Here ----
+    response = client.delete("/goals/1")
+    response_body = response.get_json()
 
     # Assert
-    # ---- Complete Assertions Here ----
-    # assertion 1 goes here
-    # assertion 2 goes here
-    # ---- Complete Assertions Here ----
+    assert response.status_code == 404
+    assert response_body == None
+    assert Goal.query.all() == []
```

Comment:
This last assert statement doesn't scale well. We can't assume that deleting a goal would always produce an empty goal table. Rather, we can do a query to retrieve a goal at id `1` and confirm that `None` is returned. 

```suggestion
    assert Goal.query.get(1) == None
```